### DISCUSSION TITLE
Unable to upload few contents

### DISCUSSION THREAD

**Discussion by paritshivani** [Sunbird-Ed/Community#682] [Bugs] [answered]
Hello team!
Hope you are doing well!

We are trying to upload few contents on our sunbird portal but getting error as file is not supported
attaching the contents here. how can we resolve this? Is there any issue with the following content?

[Module 1 – Processing and Treatment Technologies.zip](https://github.com/Sunbird-Ed/Community/files/14189360/Module.1.Processing.and.Treatment.Technologies.zip)

@rajeevsathish , @vinukumar-vs , @komalm

**Comment by rajeevsathish** [comment]
@paritshivani the link is redirecting to the discussion only 
we need the content type and also the error that you are getting can you please help us on this

cc: @vinukumar-vs

**Reply by paritshivani** [reply]
Hello @rajeevsathish ,
I have updated the ticket link . Now you will be able to download content.
Content type is of HTML5 and Js

**Reply by paritshivani** [reply]
Hello team,
Any updates on this?

**Comment by rajeevsathish** [comment]
@vinukumar-vs can you please look into the zip file and see if this is correct that we support the zip 

cc: @mohitga

**Reply by vinukumar-vs** [reply]
This zip file should be `index.html` in the root folder. But the attached zip doesn't have this index.html file. Hence it will fail.

You can rename story.html or index_lms.html to `index.html` and try to upload.
<img width="243" alt="image" src="https://github.com/Sunbird-Ed/Community/assets/4496096/25ca7a77-dbdc-4f53-be7d-855575c958cf">

@rajeevsathish @mohitga [@amitpriyadarshi](https://github.com/amitpriyadarshi)

**Reply by Reshma-mahadik** [reply]
@vinukumar-vs   I have attempted to rename both html files to index.html alternatively,but unfortunately,it did't resolve the issue.

Could you please provide further guidance or any alternative solutions to resolve this issue? I appreciate your assistance in troubleshooting this matter.

**Reply by vinod303** [reply, marked as answer]
@Reshma-mahadik ,
Before uploading / creating html zip file, first thing to do is to see, whether it is working locally. ie. using your browser, you should be able to point to the index.html in that folder, and then it should work.
If it doesnt, then UPLOADING  the same also gives issues.
And, as suggested in the thread, there must be an index.html  present. 

ex. The HTML Folder should have the following structure
1. Css (folder)
2. Js (folder)
3. Images (folder)
4. Soundfile (folder)
5. index.html
Once you have the above html structure in place and tested locally, select all these files and zip these contents as one zip file.
(Note: please do not zip the parent folder where these files stored.)
And then upload to Sunbird(workspace->create->upload)

**Reply by Reshma-mahadik** [reply]
@vinod303  sir Thank you for providing the solution. We've implemented the steps you outlined, resulting in the resolution of the issue.

**Reply by vinod303** [reply]
@Reshma-mahadik  great!
pls mark this is answered and close this ticket
